de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/editor/LocalEditorStatusChangeHandler.java

```diff
@@ -1,5 +1,6 @@
 package de.fu_berlin.inf.dpp.intellij.editor;
```

Comment:
As stated in the commit message, the logic added to this handler will be moved into a separate handler in an upcoming commit that moves all the Intellij event handler into a separate sub-package.